### ISSUE TITLE
Provide volume replica mode details in mayactl 

### ISSUE THREAD

**Issue by ksatchit** [openebs/openebs#1209]
## Is this a BUG REPORT or FEATURE REQUEST?
FEATURE REQUEST

**What happened**:

There is a need to obtain the replica write status (```mode```) , generally after voluntary/involuntary restarts. Currently, the only way to get this is to manually make a REST call via curl to the jiva controller, as shown in the following snippet, We should make this info available in a mayactl command, say, ```maya volume stats``` . Currently, the volume replica status shown in this command indicates whether the replica pods are in running state OR not and this may not be the best indicator of a useful replica. 

```
karthik@MayaMaster:~$ curl GET http://10.44.0.2:9501/v1/replicas | grep createTypes | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   162  100   162    0     0     23      0  0:00:07  0:00:06  0:00:01    48
100   971  100   971    0     0   220k      0 --:--:-- --:--:-- --:--:--  220k
{
  "createTypes": {
    "replica": "http://10.44.0.2:9501/v1/replicas"
  },
  "data": [
    {
      "actions": {
        "preparerebuild": "http://10.44.0.2:9501/v1/replicas/dGNwOi8vMTAuNDQuMC4xOjk1MDI=?action=preparerebuild",
        "verifyrebuild": "http://10.44.0.2:9501/v1/replicas/dGNwOi8vMTAuNDQuMC4xOjk1MDI=?action=verifyrebuild"
      },
      "address": "tcp://10.44.0.1:9502",
      "id": "dGNwOi8vMTAuNDQuMC4xOjk1MDI=",
      "links": {
        "self": "http://10.44.0.2:9501/v1/replicas/dGNwOi8vMTAuNDQuMC4xOjk1MDI="
      },
      "mode": "RW",
      "type": "replica"
    },
    {
      "actions": {
        "preparerebuild": "http://10.44.0.2:9501/v1/replicas/dGNwOi8vMTAuMzYuMC4xOjk1MDI=?action=preparerebuild",
        "verifyrebuild": "http://10.44.0.2:9501/v1/replicas/dGNwOi8vMTAuMzYuMC4xOjk1MDI=?action=verifyrebuild"
      },
      "address": "tcp://10.36.0.1:9502",
      "id": "dGNwOi8vMTAuMzYuMC4xOjk1MDI=",
      "links": {
        "self": "http://10.44.0.2:9501/v1/replicas/dGNwOi8vMTAuMzYuMC4xOjk1MDI="
      },
      "mode": "RW",
      "type": "replica"
    }
  ],
  "links": {
    "self": "http://10.44.0.2:9501/v1/replicas"
  },
  "resourceType": "replica",
  "type": "collection"
}
```

**What you expected to happen**:

*maya volume stats* or any equivalent command should provide replica RW mode.

**Comment by kmova** [comment]
The mayactl code between the v0.5 and 0.6 branch has been refactored quite a bit. This issue will be fixed in 0.6

With 0.5.x version, the curl command to controller can be used to get the details.

**Comment by utkarshmani1997** [comment]
@ksatchit this feature is merged in this PR, so close this issue.
